### DISCUSSION TITLE
Compiling Audiogridder on Mac M1

### DISCUSSION THREAD

**Discussion by Doepfer76** [apohl79/audiogridder#1168] [General]
Hi,
I am not sure if this the right place to post this, but any help would be greatly appreciated.
I downloaded the Audiogridder code from github and i am trying to build it on an M1 Mac for Arm64 and X86_64.
I am stuck linking statically the FFMPEG libraries.
I installed FFmpeg via homebrew. I was able to link them dynamically for Arm64 by changing CMakeLists.txt but I can't link them statically without errors.
To eliminate errors i added
find_package(SDL2 REQUIRED)
find_package(Fontconfig REQUIRED)
find_package(Freetype REQUIRED)
find_package(OpenGL REQUIRED)
find_package(BZip2 REQUIRED)
find_package(FFmpeg REQUIRED)
find_package(WebP REQUIRED)

but now i am stuck here:

Undefined symbols for architecture arm64:
  "_TessBaseAPIAllWordConfidences", referenced from:
      _filter_frame in libavfilter.a(vf_ocr.o)

Is it possible to provide a link to download the static universal binary libraries?
Thank you very much!
Andy

**Comment by apohl79** [comment]
Please follow https://github.com/apohl79/audiogridder/blob/master/DEVELOP.md.
